pkgs/tools/security/der-ascii/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule {
+  pname = "der-ascii";
+  version = "unstable-2022-01-18";
```

Comment:
The upstream has been convinced to tag a 0.1.0 release (some caveats w.r.t semver expectations discussed in this thread: https://github.com/google/der-ascii/issues/23)